File: Backend/apptienda/src/main/java/com/iesagora/jesus/apptienda/business/dto/ProductoCarritoDTO.java
package com.iesagora.jesus.apptienda.business.dto;

import com.iesagora.jesus.apptienda.business.model.CategoriaRopa;
import com.iesagora.jesus.apptienda.business.model.Talla;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class ProductoCarritoDTO {
	
    private String nombreProducto;
    private Talla talla;    
    private String color;
    private CategoriaRopa categoria;
    private String imagenProducto;

}

File: Frontend/apptienda/src/app/model/producto-carrito.dto.ts
import { CategoriaRopa } from "./categoria.enum";
import { Talla } from "./talla.enum";

export interface ProductoCarritoDTO {
    nombreProducto: string;
    talla: Talla;
    color: string;
    categoria: CategoriaRopa;
    imagenProducto: string;
}